mmv1/templates/terraform/examples/healthcare_fhir_store_basic.tf.erb

```diff
@@ -8,6 +8,7 @@ resource "google_healthcare_fhir_store" "default" {
   disable_referential_integrity  = false
   disable_resource_versioning    = false
   enable_history_import          = false
+  enable_history_modifications   = false
```

Comment:
As it is beta only field, can you please move it to another example file https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/examples/healthcare_fhir_store_notification_configs.tf.erb ?

Reply:
Done. Thanks!